packages/wire/__tests__/messages/ClosingSignedMessage.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { ChannelId } from "@node-lightning/core";
+import { ChannelId, Value } from "@node-lightning/core";
```

Comment:
Sample refactoring of ClosingSignedMessage, kindly check if going in right direction?

Reply:
Yep, lookin good!